x-pack/test/functional_embedded/tests/index.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { FtrProviderContext } from '../ftr_provider_context';
+
+export default function ({ loadTestFile }: FtrProviderContext) {
+  describe('Kibana embedded', function () {
+    this.tags('ciGroup6');
```

Comment:
https://kibana-ci.elastic.co/job/elastic+kibana+master/5930/flowGraphTable/

Looks like that group takes 44 min, how about using `ciGroup2` that is 34 min at the moment ?